src/scss/common/careers.scss

```diff
@@ -1,82 +1,129 @@
 .career {
-  margin: 2rem 0 0 0;
-  width: 100%;
-  .video-column {
-    background-color: black;
-    padding: 0;
-    min-height: 300px;
-    min-width: 500px;
+  &-section {
+    padding: 2rem 0;
+    width: 100%;
+    background-color: #ffdf80;
   }
 
-  .video-description-column {
-    background-color: white;
-    padding: 2rem;
-    align-content: center;
+  &-background-img {
+    background-image: url("../../../public/assets/CareerBackground.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+    background-size: contain;
   }
-}
 
-.career-title {
-  background-color: black;
-  color: white;
-  margin-bottom: 0;
-  padding: 0.5rem 2rem;
-  border-radius: .25rem;
-  text-align: center;
-}
+  &-title {
+    color: #120540;
+    font-size: 40px;
+    font-family: "dm-sans-light", DMSans-Light, sans-serif;
+    padding: 20px 0;
+  }
+
+  &-subtitle {
+    color: #120540;
+    font-size: 24px;
+    font-family: "dm-sans-medium", DMSans-Medium, sans-serif;
+    padding: 20px 30px;
+  }
 
-.cs-journey {
-  display: flex;
-  margin: 2rem 0 0 0;
-  .left-column {
-    flex-grow: 1;
-    min-width: 322px;
-    max-width: calc(100% - 322px);
-    padding: 2rem;
+  &-text {
+    color: #1c1c1c;
+    font-size: 19px;
+    font-family: "roboto", Roboto, sans-serif;
+    padding: 10px 30px;
+  }
+
+  &-media-row {
+    display: flex;
+    justify-content: space-between;
+    gap: 100px;
+    flex-wrap: wrap;
+  }
+
+  &-media-row > .col,
+  &-text-row > .col {
```

Comment:
are all these styles necessary with the flexing on the column? ReactStrap should be doing the vast majority of this for us right?

Reply:
You are right we don't need those, in fact removing them actually aligned everything a bit better